packages/data/src/registry.js

```diff
@@ -347,6 +273,7 @@ export function createRegistry( storeConfigs = {} ) {
 	}
 
 	let registry = {
+		namespaces,
```

Comment:
I recall in early iterations of plugins implementation I needed to consider in `withPlugins` whether the result of `mapValues` should be left alone if not a function. Here, `namespaces` is being added and is unique from the others in that it is an object, not a function. It cannot therefore be `apply`'d.